src/layouts/indexPageModes.js

```diff
@@ -0,0 +1,52 @@
+import Header from '../components/header/Header'
+import Container from '../components/container/Container'
+import Description from '../components/description/Description'
+
+export class indexPageModes {
+    constructor() {
+        this.container = null        
+    }    
+
+    render() {
+        if (!this.container) {
```

Comment:
Currently `constructor` and this condition does not seem to be necessary. Do you have plans to add an option to set a container in the `constructor` in the future?